docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,680 @@
+
+. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials. Copy the `create_service_account` module and paste it into the `rhoas_kafka.yml` file to create a service account.
+. Enter values for the `name` and `short description` fields. When you run the `create_service_account` module as part of the playbook at the end of this procedure, Ansible populates the generated service account credentials in the `client_id` and `client_secret` fields in the terminal after it creates the service account.
++
+.Example `create_service_account` module
+[source,yaml]
+----
+- name: Create Service Account
+    create_service_account:
+      name: "service-account-name"
+      description: "This is a description of the service account"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      srvce_acc_resp_obj
+----
+
+. Copy the contents of the `create_kafka_acl_binding` module and paste it into the `rhoas_kafka.yml` file to create an ACL for the service account with some default values, and bind that ACL to the Kafka instance and service account.
+. You can allow Ansible to get the Kafka ID from the `kafka_req_resp` variable or enter it in the `kafka_id` field.
++
+The following list is a description of each field that must have a value in an ACL binding module.
+
+`principal`:: The user that this binding applies to. This example uses the service account client ID.
+`resource_name`:: The name of the resource you want access to. This example uses the name that is passed when creating the topic.
+`resource_type`:: The type of resource you want access to. This example uses *Topic*.
+`pattern_type`:: The type of pattern of the ACL. The below example uses the `PREFIXED` pattern type meaning that Kafka will try to match the prefix of the resource name with the resource specified in the ACL.
+`operation_type`:: The type of operation (an action performed on a resource) that is allowed for the given user on this module.
+`permission_type`:: Whether permission is given to the user or taken away.
++
+.Example `create_kafka_acl_binding` module
+[source,yaml]
+----
+- name: Create kafka ACL Service Binding
+    rhoas.rhoas.create_kafka_acl_binding:
+      kafka_id: "{{ kafka_req_resp.kafka_id }}"
+      # To hard code the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+      principal: " {{ srvce_acc_resp_obj['client_id'] }}"
+      # To hard code the principal_id, uncomment and use the following line:
+      # principal: "PRINCIPAL_ID"
+      resource_name: "topic-name"
+      resource_type: "Topic"
+      pattern_type: "PREFIXED"
+      operation_type: "all"
+      permission_type: "allow"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register: kafka_acl_resp
```

Comment:
Similar to my comments about the service account module, I'd show the example module first. Then, as a step that follows, describe the things a user must specify in it.